### ISSUE TITLE
Library in JAR format?

### ISSUE THREAD

**Issue by MarcoLizza** [timroes/EnhancedListView#26]
Hi, have you considered providing the library in JAR format, too?

I'm still developing using Eclipse and I would like _not_ to include your source file (and resources) in the app-codebase itself...

**Comment by timroes** [comment]
I am sorry, I won't. Please also see here: https://github.com/timroes/EnhancedListView/issues/22#issuecomment-31536040

**Comment by MarcoLizza** [comment]
Thanks, didn't noticed the other issue.

**Comment by devilmac** [comment]
Hi timroes,

I was able to add your library in my Eclipse project by doing the following:
- in build.gradle file inside the library folder (not the root folder project), I added this line: **apply plugin: 'eclipse'**;
- I had to add the ANDROID_HOME environment variable pointing to the root of Android SDK (it needs to reboot the system before it works; if this wouldn't work, add to the root folder project a **local.properties** file containing this line: **sdk.dir=pathToAndroidSDK**);
- in a terminal, I have executed this command from the root folder of the project:
  **gradle eclipse**.

When the process is finished, inside the library folder there were these three files:
- **.classpath** file;
- **.project** file;
- **.settings** folder.

In other words, the command above has "translated" the gradle project into an Eclipse project.

Done that, I have imported the project into my Eclipse workspace, made changes to get a valid project without any error, set it to be an Android library and added it to my application as usual in Eclipse.

I hope this will help.
